Hymns/Common/PDFViewer.swift

```diff
@@ -13,7 +13,7 @@ struct PDFViewer: UIViewRepresentable {
 
     func updateUIView(_ pdfView: PDFView, context: Context) {
         guard let url = url else {
-            // TODO create error state
+            displayErrorState(pdfView)
             return
```

Comment:
I think url can just straight up be non-optional, and everything should still work

Reply:
The URL type is always an optional value. I don't even think it is possible to store a URL without an optional. Can I just do `guard let url = url else {
return } ` and leave it at that